python/adelphi/setup.py

```diff
@@ -13,17 +13,39 @@
 if not PY3:
     dependencies.append('backports.tempfile ~= 1.0')
 
+long_description = ""
+with open("README.md") as f:
+    long_description = f.read()
+
 setup(
-   name='adelphi',
-   version='0.1.0',
-   author='Gianluca Righetto',
-   author_email='gianluca.righetto@datastax.com',
+    name='adelphi',
+    url='https://github.com/datastax/adelphi',
+    description='Tooling for Adelphi',
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    version='0.1.0',
+    project_urls={
+        'Source': 'https://github.com/datastax/adelphi',
+        'Issues': 'https://github.com/datastax/adelphi/issues',
+    },
+    author='DataStax',
+    classifiers=[
+        'Development Status :: 3 - Alpha',
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: Apache Software License',
+        'Natural Language :: English',
+        'Operating System :: OS Independent',
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
+        'Programming Language :: Python :: 3.9',
+        'Programming Language :: Python :: Implementation :: CPython',
```

Comment:
python-driver includes pypy support here.  We _should_ be fine on pypy but I haven't tested it explicitly (and didn't want to introduce that as an ongoing thing) so for now I just dropped it.